x-pack/plugins/cases/server/client/cases/bulk_get.ts

```diff
@@ -88,12 +88,21 @@ const constructErrors = (
   const errors: CasesBulkGetResponse['errors'] = [];
 
   for (const soError of soBulkGetErrors) {
-    errors.push({
-      error: soError.error.error,
-      message: soError.error.message,
-      status: soError.error.statusCode,
-      caseId: soError.id,
-    });
+    if (isSODecoratedError(soError.error)) {
+      errors.push({
+        error: soError.error.output.payload.error,
+        message: soError.error.output.payload.message,
+        status: soError.error.output.statusCode,
```

Comment:
I was wondering if this should be `payload.statusCode` but I just realized the level above also has the `statusCode` when `isBoom` 😞

Reply:
Yes, the `Boom` error has the `statusCode` in the `output` but the `error` and `message` in the payload.